resources/validations/README.md

```diff
@@ -17,23 +17,36 @@ project structure
 To validate xml files using schematron
 ---
 
-example
+*Prerequesite
+if you haven't done it previously: to add the needed dependencies (declared by .gitmodules), run the following:*
+
+`-f` *\<required>* is the input file to be tested. ex: `-f test/demo/FedRAMP-SSP-OSCAL-Template.xml`
 
-`./validate_with_schematron.sh test/demo/FedRAMP-SSP-OSCAL-Template.xml`
+`-s` *\<optional>* schematron directory used to validate the file. Each .sch found will be compliled and generate a separate report. defaults to src/*.sch relative to this script.  ex: `-o ~/mySchematronDirectory`
 
-you must pass in a file name you want validated as argument `$1`. by default it will compile and validate the input with all `src/*.sch` files.
+`-o` *\<optional>* is an the root of the report output. ex: `-o ~/dev`
 
-if you wish to override the default version (currently 10.2) of `SAXON HE`, you may pass it as the argument `$2`
+`-v` *\<optional>* if you wish to override the default version (currently 10.2) of `SAXON HE`, that is downloaded and used if $SAXON_CP is not specified. ex:  `-v 10.2.2`
```

Comment:
Perhaps mention if `$SAXON_CP` is set, then it will fail as it refuses to choose which one takes priority, or so it seems from testing.

Reply:
correct, and I will add that to doc